lib/UserBackend.php

```diff
@@ -118,41 +120,41 @@ protected function userExistsInDatabase(string $uid): bool {
 	 * @param array $attributes
 	 */
 	public function createUserIfNotExists(string $uid, array $attributes = []): void {
-		if (!$this->userExistsInDatabase($uid)) {
-			$values = [
-				'uid' => $uid,
-			];
-
-			// Try to get the mapped home directory of the user
-			try {
-				$home = $this->getAttributeValue('saml-attribute-mapping-home_mapping', $attributes);
-			} catch (\InvalidArgumentException) {
-				$home = '';
-			}
+		if ($this->userExistsInDatabase($uid)) return;
```

Comment:
```suggestion
		if ($this->userExistsInDatabase($uid)) {
			return;
		}
```

Returning early is fine, but we always use `{ }` blocks, even if there is just one line of code (defensive approach).